### PR TITLE
Project create external

### PR DESCRIPTION
**Pull request by DavidFair**

### CODE REVIEW

**Comment by DavidFair** [comment]
TODO:
- [ ] Port Router add Interface

File: actions/src/subnet_actions.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        return func(**kwargs)
+
+    def subnet_show(self, subnet):
+        """
+        Show subnet properties
+        :param subnet: Name or ID
+        :return: (status (Bool), reason/output (String/Dict))
+        """
+        try:
+            subnet = self.conn.network.find_subnet(subnet, ignore_missing=False)
```
Comment (Reillyhewitson) [inline actions/src/subnet_actions.py:30]: I don't see a conn variable being created here. How is it being accessed?

File: actions/router.delete.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 description: Delete Router
 enabled: true
```
Comment (Reillyhewitson) [inline actions/router.delete.yaml:3]: As this isn't implemented you might want to set this and router.update to false.

File: actions/src/subnet_actions.py (hunk 1 above)
Comment (DavidFair) [reply actions/src/subnet_actions.py:30]: This is the legacy way of using the API, I'll add a `raise NotImplementError` to make it clear :+1:

**Review by Reillyhewitson — APPROVED** [review]

**Pull request merged**